### PR TITLE
Replace Saucelabs with Browserstack

### PR DESCRIPTION
**Pull request by bengummer**
Things disabled as part of this PR:
1. Linux tests (not supported by BrowserStack)
2. Opera and Android Chrome (tests failing, browser commented out for now)
3. IE 8/9/10 (not supported, was enabled previously but commented out now)

Note that we'll need to add the `BROWSERSTACK_USERNAME` and `BROWSERSTACK_KEY` env vars to Travis - can get these from 'Automated' section of https://www.browserstack.com/accounts/settings

Once this is released to npm, we'll need to bump any skatejs/named-slots repos which depend on this. They use fixed npm versions of `skatejs-build` so builds shouldn't break when this is released.

### CODE REVIEW

**Comment by treshugart** [comment]
lgtm

File: browserstack.browsers.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  },
+
+  // IE (IE 8/9/10 not supported)
+  internet_explorer_11: {
+    browser: 'ie',
+    os: 'WINDOWS',
+    os_version: '8.1',
+    browser_version: '11',
+  },
+  // internet_explorer_10: {
```
Comment (adevnadia) [inline browserstack.browsers.js]: Maybe delete them at all? Git remembers everything

File: browserstack.browsers.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  // },
+
+  // Edge
+  microsoftedge_latest: {
+    browser: 'edge',
+    os: 'WINDOWS',
+    os_version: '10',
+  },
+
+  // Opera (disabled for now)
```
Comment (adevnadia) [inline browserstack.browsers.js]: same here

File: browserstack.browsers.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  // },
+
+  // iOS
+  iphone: {
+    os: 'ios',
+    os_version: '9.1',
+    device: 'iPhone 6S',
+  },
+
+  // Android (disabled for now)
```
Comment (adevnadia) [inline browserstack.browsers.js]: same here

**Comment by adevnadia** [comment]
LGTM

**Pull request merged**